modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -305,6 +305,11 @@ namespace dnn //! This namespace is used for dnn module functionlaity.
      */
     CV_EXPORTS Ptr<Importer> createTensorflowImporter(const String &model);
 
+    /** @brief Reads a network model stored in TensorFlow model files.
+      * @details This is shortcut consisting from createTensorflowImporter and Net::populateNet calls.
+      */
+    CV_EXPORTS_W Net readNetFromTensorflow(const String &model);
```

Comment:
@alalek Tests doesn't pass because occur the builds errors in python binding. Could you advise me about it?